### DISCUSSION TITLE
How the OBS’s team will handle the Arm release of OBS 28 on MacOS?

### DISCUSSION THREAD

**Discussion by AndreaCicca** [obsproject/obs-studio#7069] [Development Questions] [answered]
- Will there be different branches that have to be selected on Steam?
- Will the team maintain separate version for OBS or can we expect an universal package?

**Comment by sebastian-s-beckmann** [comment, marked as answer]
We will most likely have separate versions on the website.
On steam, afaik there will be an automatic detection that launches the correct version, though both will be downloaded. This should be transparent to the user.
